pkgs/games/liberal-crime-squad-git/default.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchFromGitHub, stdenv, ncurses, autoconf, automake , SDL2, SDL2_mixer }:
+
+stdenv.mkDerivation rec {
+  version = "2016-03-03";
+  name = "liberal-crime-squad-git-${version}";
```

Comment:
Hm, indeed, I haven't known that we have such a line in the manual. However, our usual practice is to name at least directories and attributes without the suffix. I filed https://github.com/NixOS/nixpkgs/issues/14572 to discuss this; meanwhile let's use your own judgement.